docs/content/misc/contribute.ngdoc

```diff
@@ -61,7 +61,8 @@ To build AngularJS, you clone the source code repository and use Grunt to genera
 minified AngularJS files:
 
 ```shell
-# Clone your Github repository:
+# Clone your Github repository(Note: Do not go with `sudo` user when cloning into this repository 
```

Comment:
We can't really provide a documentation for unix-specific commands. If somebody uses `sudo` to clone the repo, they should know what they are doing.